src/main/java/org/broadinstitute/hellbender/engine/ReadShard.java

```diff
@@ -147,9 +160,17 @@ public boolean containsStartPosition( final Locatable loc ) {
      */
     @Override
```

Comment:
The javadoc for `setDownsampler` already states that `Downsampling will be performed after any requested read filtering.` -- do you want this comment copied anywhere else?